line-sdk/src/main/java/com/linecorp/linesdk/openchat/OpenChatParameters.java

```diff
@@ -0,0 +1,83 @@
+package com.linecorp.linesdk.openchat;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class OpenChatParameters {
+    private static int DEFAULT_CATEGORY_ID = 17; // Game
+    private OpenChatParameters() {}
+
+    String name;
+    String description;
+    String creatorDisplayName;
+    int categoryId = DEFAULT_CATEGORY_ID;
+    Boolean isSearchable = true;
+
+    public String toJsonString() {
+        JSONObject jsonObject = new JSONObject();
+
+        try {
+            jsonObject.put("name", name);
+            jsonObject.put("description", description);
+            jsonObject.put("creatorDisplayName", creatorDisplayName);
+            jsonObject.put("category", categoryId);
+            jsonObject.put("allowSearch", isSearchable);
+        } catch (JSONException exception) {
+            return "{}";
+        }
+
+        return jsonObject.toString();
+    }
+
+    public static class Builder  {
+        private OpenChatParameters openChatParameters = new OpenChatParameters();
```

Comment:
In Builder Pattern, the object creation is usually created in the `build()` or `create()` method. I think we need to move this to `build()` method. Please refer to https://refactoring.guru/design-patterns/builder/java/example#example-0--builders-CarBuilder-java.

Reply:
Using builder here is for checking parameter correctness, and allows developer knowing how to construct OpenChatParameter. Refactored it into kotlin, so Builder is no longer necessary. please check https://github.com/line/line-sdk-android/pull/45/commits/3bb388dc69b8f8096b6816ed1738c9dfd3fc6097